src/ui-setup/scripts/generate-ui.js

```
@@ -37,8 +41,20 @@ const renderers = [
   [renderIndex, INDEX_PATH]
 ]
 
-ncp(SOURCE_PATH, DESTINATION_PATH, err => {
-  if (err) {
+fse.copy(SOURCE_PATH, DESTINATION_PATH, err => {
+  //if cannot find happ dir, create one and signal error
+  if (err && err[0].errno === -2) {
+    exec('[ ! -d ./happ ] && mkdir ./happ', (error, stderr) => {
+      if (error) {
+        console.error(`exec error: ${error}`)
+      } else if (stderr) {
+        console.error(`stderr: ${stderr}`)
+      } else {
+        console.error("Error: Missing happ directory.\n\nNote: A new happ directory has now been pre-populated for you.  Please run 'ui:generate' once again to generate the ui.")
+      }
+    })
+    return
```

if `happ` directory is just missing, this should just throw an error